pytensor/compile/__init__.py

```diff
@@ -30,6 +30,7 @@
     OPT_O3,
     OPT_STABILIZE,
     OPT_UNSAFE,
+    PYTORCH,
```

Comment:
this can be removed I think

Reply:
Why?

Reply:
I can keep it, but I did not call the inner rewrites (ala [numba](https://github.com/pymc-devs/pytensor/blob/main/pytensor/link/numba/dispatch/basic.py#L437)). I can if you think that's needed!

Reply:
We should, but anyway that's not the only reason why that exists.